mne/preprocessing/ica.py

```diff
@@ -1066,7 +1066,7 @@ def _find_bads_ch(self, inst, chs, threshold=3.0, start=None,
                 idx_unique.remove(i)
         if len(scores) == 1:
             scores = scores[0]
-        labels[prefix] = list(idx)
+        labels = list(idx)
```

Comment:
You can now delete the `labels = {}` on line 1029, and the `labels[...]` line on 1053. This is weird, though, since it seems like someone went through effort to add these lines, and now they are not used...?

Reply:
Ok, I know what this is. That was a casualty of the refactoring of three find_bads functions into one private function, _find_bads_ch in #5807. The original functions put both a general entry in the ICA.labels_ for e.g. all EOG/ECG channels, and then specific entries for specific EOG/ECG channels. This more specific labelling was lost in the refactoring. I have now restored it - it is implemented by the private _find_bads_ch, while the more general labelling is done by the relevant public functions.

Reply:
>  That was a casualty of the refactoring of three find_bads functions into one private function, find_bads_ch in #5807. 

Can you add a test that would have caught this regression? Probably that there is some particular key in `ica.labels_` after some ica-find-bads operation?